packages/confirmation-modal/src/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -112,17 +122,19 @@ export const ConfirmationModal = React.forwardRef(
         </div>
         <Footer>
           <Button
+            {...confirmButtonProps}
+            data-testid={LGIDS_CONFIRMATION_MODAL.confirm}
+            disabled={!confirmEnabled || isConfirmDisabled}
+            className={cx(buttonStyle, confirmButtonProps?.className)}
             variant={variant}
-            disabled={!confirmEnabled || submitDisabled}
             onClick={handleConfirm}
-            className={buttonStyle}
-            data-testid={LGIDS_CONFIRMATION_MODAL.confirm}
           >
-            {buttonText}
+            {buttonText || confirmButtonProps?.children || 'Confirm'}
```

Comment:
should this follow the same fallback logic order as above?
`confirmButtonProps?.children || buttonText || 'Confirm'`

Reply:
I think that `buttonText` should be first since we don't want to break the current functionality.

Reply:
oh good catch, i'll swap those